src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -149,18 +150,20 @@ void CHyprGroupBarDecoration::draw(CMonitor* pMonitor, float a) {
 
         CColor            color = m_dwGroupMembers[WINDOWINDEX].lock() == g_pCompositor->m_pLastWindow.lock() ? PCOLACTIVE->m_vColors[0] : PCOLINACTIVE->m_vColors[0];
         color.a *= a;
-        g_pHyprOpenGL->renderRect(&rect, color);
 
         rect = {ASSIGNEDBOX.x + floor(xoff) - pMonitor->vecPosition.x + m_pWindow->m_vFloatingOffset.x,
                 ASSIGNEDBOX.y + ASSIGNEDBOX.h - floor(yoff) - ONEBARHEIGHT - pMonitor->vecPosition.y + m_pWindow->m_vFloatingOffset.y, m_fBarWidth,
                 (*PGRADIENTS || *PRENDERTITLES ? *PHEIGHT : 0)};
+        
+        g_pHyprOpenGL->renderRect(&rect, color, *PGROUPROUNDING); 
```

Comment:
why has this moved?

Reply:
don't think it's redundant since the value of `rect` will be different
additionally this is for the thin bar at the bottom of the bar so rounding it without checking if `2 * rounding <= box.(h/w)` will likely make it look ugly

Reply:
the proper way to handle the thin bottom bar would be to offset the `rect.x` by `rounding` and reducing the `rect.w` by `2 * rounding` and no rounding, so the main bar would be rounded and thin bar at the bottom would have the same size as the main bar at the bottom